cmd/challenge-api/main.go

```diff
@@ -46,25 +48,29 @@ func connect(mysql app.MysqlConfig) *sql.DB {
 // @BasePath /
 
 func main() {
+	log := logrus.New()
 
+	logEntry := logrus.NewEntry(log)
 	config := app.ReadConfig(".env")
+	log.SetOutput(os.Stdout)
 
-	db := connect(config.MysqlConfig)
+	db, logdb := connect(config.MysqlConfig)
+	logEntry.Info(logdb)
 
 	err := mysqldb.RunMigrations(config.MysqlConfig.URL())
 	if err == nil {
-		fmt.Println("Migrations ok!")
+		logEntry.Infoln("migrations ok!")
 	}
 	if err != nil {
-		log.Fatal("Error in db migrations! ", err)
+		logEntry.Fatal(fmt.Errorf("error in db migrations! %s", err.Error()))
```

Comment:
I had thought that maybe the messages were too generic and that it might make the log message difficult to understand, but I forced some errors to see the messages and it really always says it's the migration. Changed 😁